starter-code/iteration2/iteration2/src/app/app.module.ts

```diff
@@ -0,0 +1,20 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+
+import { AppComponent } from './app.component';
+import { IterationMainComponentComponent } from './iteration-main-component/iteration-main-component.component';
+import { IterationCommentsComponentComponent } from './iteration-comments-component/iteration-comments-component.component';
+
+@NgModule({
+  declarations: [
+    AppComponent,
+    IterationMainComponentComponent,
```

Comment:
You don't need to name your components with the ```component```word cause it adds it by itself, otherwise you end up with the repeated word in a lot of stuff